### PR TITLE
pantheon-tweaks: init at 1.0.1

### LINKED ISSUE #115222: Package request: pantheon-tweaks
**Project description**
Pantheon Tweaks is a Switchboard plug allowing users to somewhat tweak Pantheon (elementary OS's DE) settings of the more hidden kind, not normally available from Switchboard. As NixOS has Pantheon packaged, it might be of use in Nixpkgs.

**Metadata**

* homepage URL: https://github.com/pantheon-tweaks/pantheon-tweaks/
* source URL: https://github.com/pantheon-tweaks/pantheon-tweaks/
* license: `stdenv.lib.meta.licenses.gpl3Only`
* platforms: same as Pantheon

Depending on Pantheon version currently in Nixpkgs, the predecessor elementary-tweaks, also GPLv3, might be of more use: https://github.com/elementary-tweaks/elementary-tweaks

### PR DESCRIPTION
**Pull request by bobby285271**
###### Motivation for this change

A switchboard plugin.

Closes #115222 (some related discussions also happened here, I did not put the package or the option in the pantheon namespace due to reasons mentioned inside). I am not sure [if we really need an option for this](https://github.com/NixOS/nixpkgs/issues/115222#issuecomment-905957324) though.

###### Things done

- [x] Tested the option under Pantheon 6.
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/modules/programs/pantheon-tweaks.nix [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  meta = {
+    maintainers = lib.teams.pantheon.members;
```
Comment (bobby285271) [inline nixos/modules/programs/pantheon-tweaks.nix:7]:
```suggestion
    maintainers = teams.pantheon.members;
```

**Comment by bobby285271** [comment]
@ofborg eval

**Comment by davidak** [comment]
>I am not sure if we really need an option for this though.

`programs.pantheon-tweaks.enable` is move user-friendly than
    `services.xserver.desktopManager.pantheon.extraSwitchboardPlugs = [ pkgs.pantheon-tweaks ];`
simply because it's shorter and easiert to understand. hide complexity

i will test the pr again

**Review by davidak — APPROVED** [review]
Changes look good

Builds, works

**Pull request merged**